extensions/3rdparty/LyricWiki/LyricFind/LyricFind.setup.php

```diff
@@ -31,12 +31,6 @@
 // LyricFind indexing
 $wgHooks['ParserBeforeStrip'][] = 'LyricFindHooks::onParserBeforeStrip';
 
-// edit permissions & view-source protection
-// @see http://www.mediawiki.org/wiki/Manual:$wgNamespaceProtection
-$wgGroupPermissions['*']['editlyricfind'] = false;
-$wgGroupPermissions['staff']['editlyricfind'] = true;
-$wgGroupPermissions['sysop']['editlyricfind'] = true;
-
 $wgHooks['AlternateEdit'][] = 'LyricFindHooks::onAlternateEdit';
```

Comment:
@wladekb nope! We stopped using Gracenote, but we VERY MUCH use LyricFind curreclty and that's how we stay licensed.

I know it's confusing because it changes between a bunch of things with similar names, but LyricFind _is_ our current sub-licensor :)